cpp/dolfinx/fem/ElementDofLayout.h

```diff
@@ -43,27 +43,12 @@ class ElementDofLayout
   /// @param[in] parent_map TODO
   /// @param[in] sub_dofmaps TODO
   /// @param[in] cell_type The cell type of the mesh.
-  /// @param[in] base_permutations The base permutations for the dofs on
```

Comment:
Is this description now in `basix` somewhere?

Reply:
Yes, there's a very thorough description there with examples: https://github.com/FEniCS/basix/blob/a633ad1b5a95f5d1fb62d68a9cd3795104bba96e/cpp/finite-element.h#L230-L307